recipes/default.rb

```diff
@@ -32,11 +32,13 @@
   cmd = 'cmd.exe'
 end
 
+chocolatey_install_script = "$wc = New-Object Net.WebClient; $wc.UseDefaultCredentials = $true; $wc.Proxy.Credentials = $wc.Credentials; $wc.DownloadString('#{node['chocolatey']['Uri']}')"
```

Comment:
This is more readabl I think and the command is slightly different. I no longer have a proxy I can test against but I have used this for years without issue. That said, if @dougireton 's script works both behind a proxy and without proxy, I'm fine with that.